gymnasium/wrappers/jax_to_numpy.py

```diff
@@ -103,6 +103,29 @@ class JaxToNumpyV0(
     Notes:
         The Jax To Numpy and Numpy to Jax conversion does not guarantee a roundtrip (jax -> numpy -> jax) and vice versa.
         The reason for this is jax does not support non-array values, therefore numpy ``int_32(5) -> DeviceArray([5], dtype=jnp.int23)``
+
+    Example:
+        >>> import gymnasium as gym
+        >>> env = gym.make("JaxEnv-vx")
```

Comment:
This will fail because `JaxEnv-vx` doesn't exist and annoying we don't have any environments that return jax data currently

Reply:
Theoretically this file will be ignored by pydoctest due to the changes in the `pyproject.toml` file.